src/data/accessibility-quiz.ts

```diff
@@ -423,7 +460,7 @@ const accessibilityQuiz = [
     Question:
       "How does adding a `lang` attribute to a webpage's `html` element help with accessibility?",
     Answer: "It helps screen readers in identifying the language of the page",
-    Distractor1: "It helps Google Translate to translate the page",
+    Distractor1: "It helps Google Translate to translate thea page",
```

Comment:
this doesn't look like it needed to be changed

```suggestion
    Distractor1: "It helps Google Translate to translate the page",
```